Clean up flutter_gallery.cmx's sandbox

sbrocket opened the pull request:
The "persistent-storage" Fuchsia component manifest feature is being replaced with "isolated-persistent-storage". See https://fuchsia-review.googlesource.com/c/fuchsia/+/250019

sbrocket commented:
@zanderso @jonahwilliams Could one of you review this? It seems I can't explicitly "request a review" without write access to the repo.

sbrocket commented:
Also, how easy/quick is it going to be roll this into fuchsia? I'm trying to eliminate all usages of "persistent-storage" quickly so I can remove it from the cmx schema and prevent new usages popping up. Is it just a matter of running update_3p_packages.py, merging the result, and manually rolling GI?

zanderso commented on examples/flutter_gallery/meta/flutter_gallery.cmx:
```diff
@@ -4,7 +4,7 @@
     "sandbox": {
         "system": [ "data" ],
-        "features": [ "persistent-storage" ],
+        "features": [ "isolated-persistent-storage" ],
```
It looks like this file may have been copied from somewhere else without thinking about what was actually needed here. I suspect that flutter_gallery doesn't need persistent storage, and so this line can be deleted. The previous line ("system": ["data"]) can probably also be deleted. The "services" can almost certainly be pruned down as well since flutter_gallery doesn't explicitly use any FIDL services, but maybe those can be pruned at a later time.

sbrocket replied:
Done

zanderso commented:
Rolling Flutter into Fuchsia is a little involved. I don't think we can guarantee that it can be done quickly, but I will try to get the autoroller unblocked tomorrow or Friday.

sbrocket committed:
Clean up flutter_gallery.cmx's sandbox

Delete unused 'persistent-storage' feature (which is being removed from
Fuchsia) and inclusion of '/system/data' in namepsace

sbrocket commented:
Today or Friday is fine. I can also temporarily remove topaz/packages/examples/flutter_gallery from topaz/packages/examples/all, which will let me go ahead and delete the "persistent-storage" feature until this rolls.
